### PR TITLE
Automation of CEPH-83571735 and CEPH-9938

### PR DESCRIPTION
**Pull request by SrinivasaBharath**
The code is to automate the following test cases-
  1. [CEPH-9938]- mon failures: while exercising the list-inconsistent[obj|snap|pg] command fail the mons
  2. [CEPH-83571735] - Corrupt obj|snap|omap to induce inconsistency and use fsck and repair to check and repair 
        inconsistency using ceph-bluestore-tool
These two test cases I included in the test case CEPH-9924.The test case CEPH-9924 is to create inconsistent objects the above test cases are related to the inconsistent objects which need to check during the mon restart and repair the objects by using the ceph-bluestore -tool.

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/rados/test_osd_inconsistency_pg.py [hunk 1]
```diff
@@ -70,6 +80,22 @@ def run(ceph_cluster, **kw):
             else:
                 log.error("Inconsistent object is not exists in the objects list")
                 return 1
+        osd_map_output = rados_obj.get_osd_map(pool=pool_name, obj=oname)
+        primary_osd = osd_map_output["acting_primary"]
```
Comment (pdhiran) [inline tests/rados/test_osd_inconsistency_pg.py:84]: Can we add logging here?

File: tests/rados/test_osd_inconsistency_pg.py [hunk 2]
```diff
@@ -70,6 +80,22 @@ def run(ceph_cluster, **kw):
...
+        # Executing the fsck on the OSD
+        bluestore_obj.run_consistency_check(primary_osd)
+        # Reparing the OSD using the bluestore tool
+        bluestore_obj.repair(primary_osd)
```
Comment (pdhiran) [inline tests/rados/test_osd_inconsistency_pg.py:88]: Let's add debug level here to print all the outputs from the CBT tool

**Review by pdhiran — COMMENTED** [review]
Looks good. We just need to add debug statements

**Comment by neha-gangadhar** [comment]
Thank you @SrinivasaBharath for combining multiple tests and implementing as workflows to cover scenarios. Looks good overall.

File: tests/rados/test_osd_inconsistency_pg.py (hunk 2 above)
Comment (SrinivasaBharath) [reply tests/rados/test_osd_inconsistency_pg.py]: The outputs are added.

File: tests/rados/test_osd_inconsistency_pg.py (hunk 1 above)
Comment (SrinivasaBharath) [reply tests/rados/test_osd_inconsistency_pg.py:84]: OSD details are added in the logs

**Comment by SrinivasaBharath** [comment]
Pass Log: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-SYW02E

**Review by pdhiran — APPROVED** [review]
Looks good

**Pull request merged**